### ISSUE TITLE
Help with data extraction

### ISSUE THREAD

**Issue by yanone** [mooniak/lanka-glyphsets#3]
Hi,

I want to include your four glyphsets definitions into `gfglyphsets`.

What's the best approach to convert (in an automated way) the four `.yaml` files into a `.glyphs` file that includes both encoded characters as well as unencoded glyphs (if any)? 

I saw two `.glyphs` files in the repository, but since they are only two and not four I was wondering whether they are up-to-date, and two are missing in any case.

In other words, if the `.yaml` files are your source-of-truth, can we develop a way to automatically convert those to `.glyphs`, either as part of your repository's CI or alternatively I can also do it on my side. If you say you'll keep the `.yaml` up-to-date always, I would want to find a way that their data can be updated in `gfglyphsets` automatically.

Cheers!

**Comment by pathumego** [comment]
@yanone finally we locked down on the glyphsets and naming. Yes, `.yaml` are our source-of-truth and we will keep them upto date. Can you help us setup a CI on this repo for generating .glyphs files? And now the yaml files contain almost all data required to validate fonts as well.

**Comment by yanone** [comment]
That’s awesome but I’ve stopped contracting for Google Fonts end of May this year. I’m doing lots of other stuff so I’m distracted.

`glyphsets` has actually been re-written in Rust by Simon meanwhile and I’ve already ceased maintaining it earlier for that reason. I’m not sure the process is still the same.

@simoncozens All I can do now is notify you that this idea exists, not really ask to take over.

What I would have done is written a converter from Pathum’s data to GF data format whatever it may be, and put that converter code somewhere. Then either update GF data whenever Pathum’s changes via cron job or something, or maybe have him run the converter and make a PR on commit or something.

That would be an excellent job for Claude Sonnet btw.
